webapp/views.py

```diff
@@ -407,7 +407,7 @@ def get_context_data(self, **kwargs):
 
         try:
             with open(mirrors_path) as rss:
-                mirrors = parse(rss).entries
+                mirrors = parse(rss.read()).entries
```

Comment:
@nottrobin we need to try this on python 2 I am not sure if it will be handled.

Reply:
Now that this is merging into "update-to-django2" branch, we presumably don't care if it works on python2.